bin/pycbc_inspiral

```diff
@@ -280,16 +284,21 @@ with ctx:
     logging.info("Overwhitening frequency-domain data segments")
     for seg in segments:
         seg /= seg.psd
-  
+
     logging.info("Read in template bank")
     bank = waveform.FilterBank(opt.bank_file, flen, delta_f,
         low_frequency_cutoff=None if opt.enable_bank_start_frequency else flow,
         dtype=complex64, phase_order=opt.order,
         taper=opt.taper_template, approximant=opt.approximant,
-        out=template_mem, max_template_length=opt.max_template_length)
-        
-    sg_chisq = SingleDetSGChisq.from_cli(opt, bank, opt.chisq_bins)                                           
-                                             
+        out=template_mem, max_template_length=opt.max_template_length,
+        load_compressed=True if opt.use_compressed_waveforms else False,
+        waveform_decompression_method=
+        opt.waveform_decompression_method if opt.use_compressed_waveforms else None,
+        waveform_decompression_precision=
```

Comment:
@ahnitz That is right. I agree that it would be better to remove the last option. But, in FilterBank I would want to read the dtype as `numpy.complex64` so that I can check if the precision is `single` and then compare it with the `precision` that I store as metadata in the compressed bank hdf file. However, `self.dtype` returns `<type 'numpy.complex64'>` . Is there a way to retrieve the string `numpy.complex64` from that?